packages/rest/test/unit/rest-application/rest-application.test.ts

```diff
@@ -37,30 +37,13 @@ describe('RestApplication', () => {
       );
     });
 
-    it('when attempting to bind servers via configuration', () => {
```

Comment:
I think we should re-write this test in another way or remove the code that deals with it if it no longer applicable. I'm thinking we create an application instance and call `app.server` after registering the rest component with `app.component` to get the `no multiple servers allowed` error.

Reply:
I think your proposed test case is covered by the one below if you're concerned about making sure it errors out when `RestComponent` is registered with `RestApplication`. If you're concerned about throwing `ERR_NO_MULTI_SERVER`, that case is covered by the test above. Or I could be misunderstanding what test you want to replace this one with.

Reply:
Yep the coverage I want is already there :-).

Reply:
:-)